tendrl/ceph_integration/objects/cluster/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+
+class Cluster(objects.CephIntegrationBaseObject):
+    def __init__(self, cluster_id=None, name=None,
+                 status=None, *args, **kwargs):
+        super(Cluster, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s'
```

Comment:
As discussed will rename this object as `Global` and cluser related details/attributes can be maintained under this.